.gitlab/kernel_matrix_testing/security_agent.yml

```diff
@@ -105,6 +105,18 @@ upload_secagent_tests_arm64:
   allow_failure: true
   stage: kernel_matrix_testing_security_agent
   rules: !reference [.on_security_agent_changes_or_manual]
+  timeout: 1h 30m
```

Comment:
why increase the timeout ? Shouldn't you reduce it so that we can retry sooner ?

Reply:
the timeout is reduced from `3h`

Reply:
oh you are right, I think I misread something